### PR TITLE
Support APNs payload extensions for `Title & Subtitle`

### PR DESCRIPTION
**Pull request by timakin**
This PR allows to send `Title` and `Subtitle` with an existing payload.

From iOS ver10, it supports Bold-styled title and subtitles to display rich notifications.

### CODE REVIEW

**Comment by cubicdaiya** [comment]
Add description to SPEC.md below.

https://github.com/mercari/gaurun/blob/master/SPEC.md#post-push

**Comment by timakin** [comment]
@cubicdaiya Thanks for ur quick check.
Added a description to SPEC about these params.

File: SPEC.md [hunk 1]
```diff
@@ -52,7 +54,9 @@ The request-body must has the `notifications` array. There is the parameter tabl
 |-----------------|------------|-----------------------------------------|--------|-------|----------------|
 |token            |string array|device tokens                            |o       |       |                |
 |platform         |int         |platform(iOS,Android)                    |o       |       |1=iOS, 2=Android|
+|title            |string      |title for notification                   |o       |       |only iOS        |
 |message          |string      |message for notification                 |o       |       |                |
+|subtitle         |string      |subtitle for notification                |o       |       |only iOS        |
```
Comment (cubicdaiya) [inline SPEC.md]: Isn't this optional? ( **title, too**)

Comment (timakin) [inline SPEC.md]: yes, title and subtitle are optional values.
If they weren't set, only `message` will be displayed.

As below, these values are optional.
https://github.com/RobotsAndPencils/buford/blob/1589c179b764ddceb204a439d9bf366f04c55f9b/payload/aps.go#L37

Comment (cubicdaiya) [inline SPEC.md]: If it is optional, update the value with `-` in `required` row.

**Comment by timakin** [comment]
@cubicdaiya 
Fixed the spec documentation.
And the position of them was wrong so changed the order.

**Comment by timakin** [comment]
@cubicdaiya 
Sorry to bother you again.
could you check the last fix?
My team is looking forward this opt 🙇

**Comment by cubicdaiya** [comment]
@timakin Sorry. As we are busy for another project, one moment, please.

**Comment by cubicdaiya** [comment]
@timakin Merged. Thanks.

**Pull request merged**